### ISSUE TITLE
Uncaught TypeError: Cannot read property 'querySelector' of null

### ISSUE THREAD

**Issue by leeyisoft** [geoffthibeau/simple-drag-drop-text#13]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.5.4
**System**: Mac OS X 10.11.3
**Thrown From**: [simple-drag-drop-text](https://github.com/mark-hahn/simple-drag-drop-text) package, v0.3.0
### Stack Trace

Uncaught TypeError: Cannot read property 'querySelector' of null

```
At /Users/leeyi/.atom/packages/simple-drag-drop-text/lib/simple-drag-drop-text.coffee:31

TypeError: Cannot read property 'querySelector' of null
    at /Users/leeyi/.atom/packages/simple-drag-drop-text/lib/simple-drag-drop-text.coffee:31:54
    at doNTCallback0 (node.js:419:9)
    at process._tickCallback (node.js:348:13)
```
### Commands

```
     -3:59.2.0 core:copy (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
     -3:10.7.0 core:confirm (input.hidden-input)
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "language-java",
      "language-hyperlink",
      "language-ruby",
      "language-ruby-on-rails",
      "language-toml",
      "language-csharp",
      "wrap-guide",
      "language-clojure",
      "language-coffee-script",
      "language-gfm",
      "keybinding-resolver",
      "language-go",
      "language-objective-c",
      "language-perl",
      "java-generator"
    ],
    "followSymlinks": true,
    "packagesWithKeymapsDisabled": [
      "atom-alignment"
    ],
    "themes": [
      "atom-light-ui",
      "eiffel-syntax"
    ]
  },
  "simple-drag-drop-text": {
    "copyKey": "ctrl"
  }
}
```
### Installed Packages

``` coffee
# User
atom-alignment, v0.12.1
copy-path, v0.4.0
css-snippets, v0.9.0
editorconfig, v1.2.4
eiffel-syntax, v0.4.0
highlight-selected, v0.11.2
language-erlang, v3.0.0
language-swift, v0.4.0
linter, v1.11.3
linter-erlc, v0.3.0
simple-drag-drop-text, v0.3.0
vim-mode, v0.64.0

# Dev
No dev packages
```

**Comment by suns-echoes** [comment]
I have found out that this error happens when you have disabled shadow root in your settings.

Just edit **"lib/simple-drag-drop-text.coffee"** file this way:

```
find: @lines = atom.views.getView(@editor).shadowRoot.querySelector
replace: @lines = atom.views.getView(@editor).querySelector
```

For me this solves the problem :)

**Comment by mark-hahn** [comment]
Can you put that in as a PR?

**Comment by suns-echoes** [comment]
Done.

update to code:

```
      @lines = atom.views.getView(@editor)
      if @lines.shadowRoot
        @lines = @lines.shadowRoot.querySelector '.lines'
      else
        @lines = @lines.querySelector '.lines'
```

**Comment by mark-hahn** [comment]
Did you follow the github docs?  You need to fork, pull, modify, push, and
then request the PR.

**Comment by ADTC** [comment]
@cmosespl Check out https://guides.github.com/activities/contributing-to-open-source/#contributing
Let's make it a PR :)

**Comment by geoffthibeau** [comment]
I believe this issue was resolved and returned shortly in #23.
